docs/simple_ir_walkthrough.md

```diff
@@ -1,513 +0,0 @@
-# Simple IR Walkthrough
-
-## Overview
-
-This walks through the process of lowering TensorFlow python to an IREE module,
-demonstrating the MLIR that exists at each stage. Many individual intermediate
```

Comment:
I'd say let's delete it and revive when later we are about to update its contents. Having it there is causing more confusion rather than help. OTOH, anything checked into Git is permanent; so we can easily get it back when necessary. :)

Reply:
If the specifics were labeled as outdated, would that help? The general lowering process is still the same and is helpful for seeing how IREE users MLIR dialects for an end-to-end compilation.

Reply:
It's hard to tell what's outdated and what's not without going through a thorough pass to clean things up. I think it causes confusion rather than help under such cases. But given you asked twice, I bringed it back and added a note at the very beginning.